src/Layers/xrRender_R2/r2.cpp

```diff
@@ -535,7 +547,7 @@ void CRender::create()
     o.dx10_minmax_sm = ps_r3_minmax_sm;
     o.dx10_minmax_sm_screenarea_threshold = 1600 * 1200;
 
-#ifndef USE_OGL
+#if defined(USE_DX9) || defined(USE_DX11)
```

Comment:
This is for DX11 only.. 
```suggestion
#if defined(USE_DX11)
```